data/hai ships.txt

```diff
@@ -94,6 +151,106 @@ ship "Flea"
 
 
 
+ship "Geocoris"
+	sprite "ship/hai geocoris"
+	attributes
+		category "Heavy Freighter"
+		"cost" 12540000
+		"shields" 13300
+		"hull" 6200
+		"required crew" 15
+		"bunks" 56
+		"mass" 810
+		"drag" 17.1
+		"heat dissipation" 0.45
+		"fuel capacity" 800
+		"cargo space" 761
+		"outfit space" 582
+		"weapon capacity" 172
+		"engine capacity" 94
+		Weapon:
+			Blast Radius: 300
+			Hit Force: 4500
+			Shield damage: 3000
+			Hull damage: 1500
+	outfits
+		Hyperdrive
+		"Hai Williwaw Cooling" 5
+		"Hai Valley Batteries"
+		"Pulse Turret" 2
+		"Pulse Cannon" 2
+		"Bullfrog Anti-Missile" 5
+		"Hai Diamond Regenerator"
+		`"Bondir" Atomic Steering`
+		"Geode Reactor" 2
+		`"Biroo" Atomic Thruster`
+	engine 64 77 1
+	engine -64 77 1
+	turret 47 -73 "Pulse Turret"
+	turret -47 -73 "Pulse Turret"
+	turret 37 6.5 "Bullfrog Anti-Missile"
+	turret -37 6.5 "Bullfrog Anti-Missile"
+	turret 0 63 "Bullfrog Anti-Missile"
+	turret 65 25 "Bullfrog Anti-Missile"
+	turret -65 25 "Bullfrog Anti-Missile"
+	gun 25 -109 "Pulse Cannon"
+	gun -25 -109 "Pulse Cannon"
+	explode "tiny explosion" 50
+	explode "small explosion" 50
+	explode "medium explosion" 50
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large"
+	description "Just a few years after the first humans discovered Hai space, the Hai knew they would eventually need a bigger freighter to match the larger human ones. What they did not expect, however, is that the result would surpass anything humanity has to offer, which is why it became such a popular ship amongst Hai and humans alike."
```

Comment:
The Geocoris isn't a popular ship, nor do any human fleets use it right now.

Suggested description:
`The introduction of humans to Hai space caused quite a stir among Hai society, especially in the economy where large human freighters were out competing with the smaller native Hai freighters. In order to prevent humans from taking over large parts of the Hai economy, the Geocoris was developed, a heavy freighter that surpassed anything humanity had to offer at the time.`

Reply:
thanks ill use it